editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -2499,9 +2499,8 @@ bool CanvasItemEditor::_gui_input_select(const Ref<InputEvent> &p_event) {
 
 	if (drag_type == DRAG_BOX_SELECTION) {
 		if (b.is_valid() && !b->is_pressed() && b->get_button_index() == MouseButton::LEFT) {
-			// Confirms box selection.
-			Node *scene = EditorNode::get_singleton()->get_edited_scene();
-			if (scene) {
+			// Confirm box selection.
+			if (Node *const scene = EditorNode::get_singleton()->get_edited_scene()) {
```

Comment:
```suggestion
			if (Node *scene = EditorNode::get_singleton()->get_edited_scene()) {
```
We also normally don't allow these kinds of statements as they are harder to read, I'd restore this